lib/coverband/configuration.rb

```diff
@@ -55,19 +59,28 @@ def s3_secret_access_key
     end
 
     def store
-      return @store if @store
-      raise 'no valid store configured'
+      @store ||= Coverband::Adapters::RedisStore.new(Redis.new(url: redis_url), redis_store_options)
     end
 
     def store=(store)
       if store.is_a?(Coverband::Adapters::Base)
```

Comment:
What do you think of removing the option of passing in a redis instance here?  Make this config option only take a store. Would really clean up this method and it seems pretty straight forward.

Reply:
yeah that was for old backward compatibility now is a good time to remove. Doing that. Thanks

Reply:
done